doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -24,7 +47,7 @@ Zoom                                  :kbd:`Alt` + right-click and drag
 Frame selected nodes                  :kbd:`F`
 Enter `Box` node (subgraph)	          :kbd:`↓`
 Leave `Box` node (subgraph)           :kbd:`↑`
-Search for nodes                      :kbd:`Ctrl` + :kbd:`F`
+Search for nodes                      :kbd:`Control` + :kbd:`F`
```

Comment:
I feel vaguely like we've talked about this before...the UI uses "Ctrl" when showing shortcuts (that's Qt's convention), and I think it's preferable for the documentation to match that. Have you had further thoughts on the matter?

Reply:
Hmm, I don't recall that conversation.

I'll need to reformat this. <kbd>Control</kbd> would have worked as a hybrid keystroke for both keyboard types, but since Gaffer OSX in fact uses <kbd>⌘</kbd>, it's not relevant. I think I'll just add an OSX column to each table, for when the OSX shortcut differs. We'll stick to <kbd>Ctrl</kbd> for Linux.

Reply:
What would you think to doing an automatic Ctrl->Apple conversion when we build the Mac docs? Or just documenting that "on Mac Ctrl means Apple"? I'm just trying to think of ways of not doubling up everything on what are already quite fiddly tables...

Reply:
The former would be great, but I wouldn't want it to hold up this PR. I agree the table formatting is silly, but since this is a reference I'd really like to avoid making users scroll up and down through the list to double-check things. Ideally, they would just go to the section they need (or be linked there by one of our devs) and have all the info right there. A split-second look-up.

It sounds like either decision is a stop-gap, though. Would the extra-wide tables bother you more than another note would bother me?